### PR TITLE
Update Vietnamese (Tiếng Việt) language

### PR DESCRIPTION
**Pull request by michioxd**

### CODE REVIEW

File: RetroBar/Languages/Tiếng Việt.xaml [hunk 1]
```diff
@@ -50,12 +62,14 @@
...
 
-    <s:String x:Key="show_hidden">Hiện thị cái biểu tưởng ẩn</s:String>
+    <s:String x:Key="show_hidden">Hiện thị biểu tưởng ẩn</s:String>
     <s:String x:Key="hide">Ẩn</s:String>
 
     <s:String x:Key="set_time">_Thay đổi Ngày/Giờ</s:String>
 
     <s:String x:Key="open_folder">&amp;Mở thư mục</s:String>
 
+    <FlowDirection x:Key="flow_direction">TráiQuaPhải</FlowDirection>
```
Comment (dremin) [inline RetroBar/Languages/Tiếng Việt.xaml:72]: This value shouldn't be localized, it should be either `RightToLeft` or `LeftToRight`. My understanding is that this translates to LeftToRight, which is the default value. If that is the case, could you please remove it? Thanks!

**Review by dremin — CHANGES_REQUESTED** [review]
Thanks, just one thing to change!

File: RetroBar/Languages/Tiếng Việt.xaml (hunk 1 above)
Comment (michioxd) [reply RetroBar/Languages/Tiếng Việt.xaml:72]: ah oke thanks!

**Review by dremin — APPROVED** [review]
Thanks!

**Pull request merged**